tests/test_models.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+# Create your tests here.
+
+class ModelTests(TestCase):
+
+    def test_manufacturer_format_str(self):
+        manufacturer = Manufacturer.objects.create(name="Test", country="Test")
+        self.assertEqual(str(manufacturer), "Test Test")
```

Comment:
The test `test_manufacturer_format_str` assumes that the `__str__` method of the `Manufacturer` model returns 'Test Test'. Ensure that the `__str__` method in the `Manufacturer` model is implemented to return this format. If not, adjust the test to match the actual implementation.